src/components/types/enums.js

```diff
@@ -323,6 +323,14 @@ export const TextAlign = {
   right: 'right'
 };
 
+export const TimeFormat = {
+  hhmmss: 'hh:mm:ss', 
```

Comment:
@grozdanov How about `kAuto` value? There is such value in native Lumin code but I cannot find it in `magic-script-components-lumin` 🤔